src/table.js

```diff
@@ -67,7 +74,7 @@ function Table(props) {
             </div>
             : 
             <div>
-                <h2 id='title' className="table-title">{props.table.title}</h2>
+                <div id='title' className="table-title">{props.table.title}</div>
```

Comment:
Is this not an h2 anymore due to squarespace overrides? Can we just be more specific on CSS to beat squarespace's selectors, or is it to much? using a header would be good for accessibility

Reply:
Good call- should be able to switch this back with our new implementation outside of the embed